test/rule-matches/no-naming-method-matches.js

```diff
@@ -69,6 +69,24 @@ describe('no-naming-method-matches', function () {
     assert.isFalse(actual);
   });
 
+  it('returns false for the listbox popup of a role=`combobox`', function () {
```

Comment:
If the suggestion above about using `popupRoles` is accepted, we should add a test that shows only `listbox` returns false and at least one other popup role returns true.

Reply:
That was supposed to be the next test, but I mixed up `modal` and `dialog`. Modal isn't an ARIA role.